sunflower-land/sunflower-land
# [FEAT] Play Testing Artefact Digging

# Description

The PR is a basic version of the artefact digging. It is set up for player testing of the feature. 

A beta tester can:

- Teleport into the digging area
- Dig items
- Start a new game

**NOTE** Other players will be moved to the edge of the dig area when you're digging. This is a basic implementation and not the final solution. Its just for now 😄 

<img width="463" alt="Screenshot 2024-07-15 at 12 21 23 PM" src="https://github.com/user-attachments/assets/d5a804f7-e769-4d86-b069-b42c575d5bf2">

<img width="559" alt="Screenshot 2024-07-15 at 12 21 35 PM" src="https://github.com/user-attachments/assets/9985491b-a6dd-4a3f-aa97-2fb744c35bfe">

<img width="558" alt="Screenshot 2024-07-15 at 12 21 42 PM" src="https://github.com/user-attachments/assets/991a86dd-6d92-488a-be2d-2650e097c5c9">

Fixes #issue

## Type of change

Please delete options that are not relevant.


# How Has This Been Tested?

1. Make sure you have a beta pass.
2. Go to Gold tooth on the beach and teleport into the dig area

# Checklist:
